src/Microsoft.AspNet.HttpOverrides/OverrideHeaderExtensions.cs

```diff
@@ -0,0 +1,32 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using Microsoft.AspNet.HttpOverrides;
+
+namespace Microsoft.AspNet.Builder
+{
+    public static class OverrideHeaderExtensions
+    {
+        /// <summary>
+        /// Uses X-Forwarded-For, X-Forwarded-Host, X-Forwarded-Proto, override headers.
```

Comment:
We *do* use Oxford commas, like it or not. However, the issue here is that that last comma is incorrect either way because the list has already ended.